tests/actions/test_core_actions.py

```diff
@@ -868,14 +868,29 @@ def test_APINotificationAction_run(self, checkpoint_result: CheckpointResult):
         )
 
     @pytest.mark.unit
-    @pytest.mark.xfail(
-        reason="Not yet implemented for this class", strict=True, raises=NotImplementedError
-    )
     def test_EmailAction_run(self, checkpoint_result: CheckpointResult):
         action = EmailAction(
-            smtp_address="test", smtp_port=587, receiver_emails="test1@gmail.com, test2@hotmail.com"
+            smtp_address="test",
+            smtp_port="587",
+            receiver_emails="test1@gmail.com, test2@hotmail.com",
```

Comment:
The implementation looks good, but I'd consider parameterizing the test to prove this works when there's no space, and when there's only one email.